.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+## Description
+
+- Describe your changes in detail.
+- Why is this change required?
+- What problem does it solve?
+- Add Notes if anything is left unclear or not completed
+
+*Example:*
+
+*PR for feat: Create Member rest api - PUT*
```

Comment:
I think here you can add the link to your pr but as comment session, because otherwise people are going to keep this examples in the pr. 😅

Consider all examples to include a real PR link or issue link.

Reply:
> Ok, I have added a link in the comment. Also, if there are some suggestions for other PRs that could be added as examples or to replace the current PR example, please add them.

I think 1 example should be enough